.pre-commit-config.yaml

```diff
@@ -49,3 +49,10 @@
       require_serial: false
       additional_dependencies: [ ]
       minimum_pre_commit_version: '0'
+
+    - id: pyupgrade
+      name: pyupgrade
+      entry: pyupgrade
+      language: system
+      types: [ python ]
+      args: [ --py37-plus ]
```

Comment:
Why did you choose _pyupgrade_?

Reply:
for upgrade python  to last version and to avoid conflicts between linters)